src/BarracksServer/Network/PacketHandler.cs

```diff
@@ -108,9 +108,13 @@ public void CB_START_BARRACK(IBarracksConnection conn, Packet packet)
 		{
 			var unkByte = packet.GetByte();
 			var str1 = packet.GetString(64); // [i373230 (2023-05-10)] Might've been added before, same as CB_LOGIN
+			var socialServers = BarracksServer.Instance.ServerList.GetSocialServers();
```

Comment:
Would you like me to keep the unrelated var separate from the reading packet values?

Reply:
Yea, otherwise your eyes might parse it all as one piece of code that looks like packet reading at a glance, when one line is something wholly different in reality. Proper grouping makes it easier to read.